boards/arm/curie_ble/board.cmake

```diff
@@ -0,0 +1,13 @@
+if(DEFINED ENV{ZEPHYR_FLASH_OVER_DFU})
+  set(BOARD_FLASH_RUNNER dfu-util)
+else()
+  set(BOARD_FLASH_RUNNER openocd)
+endif()
+
+set(BOARD_DEBUG_RUNNER openocd)
+
+board_runner_args(dfu-util "--pid=8087:0aba" "--alt=ble_core")
+board_runner_args(openocd --cmd-pre-load "targets 8" "--gdb-port=3334")
+
+include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
```

Comment:
Just use the include order of these files to set your defaults.

By including openocd.board.cmake first, you automatically set the flash and debug runners to `openocd`, unless a previous setting is already defined.

Users are free to override at cmake or west invocation time.

Reply:
understand, but it should be consistent, `arc/arduino_101_sss/board.cmake` and `x86/arduino_101/board.cmake` are already different